docs/big-data-cluster/deployment-guidance.md

```diff
@@ -64,6 +64,11 @@ kubectl config view
 
 After you have configured your Kubernetes cluster, you can proceed with the deployment of a new SQL Server big data cluster. If you are upgrading from a previous release, please see [How to upgrade [!INCLUDE[big-data-clusters-2019](../includes/ssbigdataclusters-ss-nover.md)]](deployment-upgrade.md).
 
+## Ensure you have storage configured
+It's important for most big data cluster deployments to have persistent storage. At this time, you need to make sure you have a plan for how you're going to provide persistent storage on the Kubernetes cluster *before* you deploy the BDC tools. 
```

Comment:
More precise, it's *before* you deploy the BDC. Tools (azdata, ADS) don't have a dependency on that.